python/ray/data/datasource/delta_sharing_datasource.py

```diff
@@ -0,0 +1,129 @@
+import logging
+from json import loads
+from typing import List, Optional, Tuple
+
+import numpy as np
+
+from ray.data._internal.util import _check_import
+from ray.data.block import BlockMetadata
+from ray.data.datasource.datasource import Datasource, ReadTask
+from ray.util.annotations import DeveloperAPI
+
+logger = logging.getLogger(__name__)
+
+
+@DeveloperAPI
+class DeltaSharingDatasource(Datasource):
+    def __init__(
+        self,
+        url: str,
+        jsonPredicateHints: Optional[str] = None,
```

Comment:
noticed some variables using the camel case naming convention. Can you update them to use snake_case? 
We follow google's python coding style guide.